proto/brpb.proto

```diff
@@ -876,7 +876,17 @@ message DeleteSpansOfFile {
     uint64 whole_file_length = 3;
 }
 
+enum MigrationVersion {
+    // The initial version. Somehow "UNKNOWN".
+    M0 = 0;
```

Comment:
nit, maybe V0 and V1? usually use V as version,

Reply:
protocol buffer requires the enum name is unique among *all* enums in the same package. The variant `V1` was used by `MetaFileVersion`...